web/auth/auth.go

```diff
@@ -130,6 +130,24 @@ func checkRedirectParam(c echo.Context, defaultRedirect string) (string, error)
 	return u.String() + "#", nil
 }
 
+func registerClient(c echo.Context) error {
+	// TODO add rate-limiting to prevent DOS attacks
+	if c.Request().Header.Get("Content-Type") != "application/json" {
+		return c.JSON(http.StatusBadRequest, echo.Map{
```

Comment:
ok let's leave it at that and add a Debt card on cleanup handling of error with jsonapi vs non-jsonapi once we have more points.